maven-core/src/main/java/org/apache/maven/utils/introspection/ReflectionValueExtractor.java

```diff
@@ -0,0 +1,325 @@
+/*
```

Comment:
same possible copyright issues as the DAG classes

Reply:
I've explained those earlier.  They come from Apache, so I think it's safe to remove the headers on the 3 files from this package.  That's what I did in a more recent commit.